worlds/noita/docs/en_Noita.md

```diff
@@ -0,0 +1,60 @@
+# Noita
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+Noita is a procedurally generated Roguelike. During runs in Noita you will find potions, wands, spells, perks and
+chests. For purposes of Archipelago multiworld functionality, collecting chests, results in an item check. Noita items 
+that can be found in other players games include, gold, max health increase, spell refresh, wands and perks. 
+Additionally, within your player settings yaml you can enable "Bad items" in the pool, resulting in negative effects on 
+the Noita world.
+
+As chests are quite rare in the Noita world, options in the Archipelago Noita mod allow you to set a kill count to spawn 
+chests. 
+
+## What is the goal of Noita 
+
+The vanilla (obvious) goal of Noita is to progress through each of the levels and fight the final boss, after defeating
+the final boss and ending the game this the credits will roll. The Archipelago implementation in Noita however, is an 
+"Endless run" continuing runs after death or even after finishing the game until all location checks are completed.
```

Comment:
```suggestion
The vanilla goal of Noita is to progress through each level and beat the final boss.
In the Archipelago implementation, however, you play an "endless run"; when you die or
beat the final boss, you continue playing from the beginning until all locations have been
checked.
```
I have never played this randomizer, and I have never played this game. This is my attempt at pruning/translating this paragraph; if I have a misconception, I got it from the original text, so it'd probably worth editing regardless.